providers/com/peerj/harvester.py

```diff
@@ -0,0 +1,56 @@
+import arrow
+from furl import furl
+
+import logging
+from share import Harvester
+
+logger = logging.getLogger(__name__)
+
+
+class PeerJHarvester(Harvester):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.base_url = 'https://peerj.com/articles/index.json'
+        self.base_preprint_url = 'https://peerj.com/preprints/index.json'
```

Comment:
It may make more sense to have a separate harvester for preprints, especially if there is the possibility of overlapping identifiers.